webapp/src/components/AddressLink/AddressLink.js

```diff
@@ -15,29 +15,39 @@ export default class AddressLink extends React.Component {
     size: PropTypes.number,
     scale: PropTypes.number,
     hasTooltip: PropTypes.bool,
-    isUser: PropTypes.bool
+    isUser: PropTypes.bool,
+    className: PropTypes.string
   }
 
   static defaultProps = {
     link: null,
-    size: 11,
+    size: 6,
     scale: 3,
-    hasTooltip: true
+    hasTooltip: true,
+    className: ''
   }
 
   render() {
-    const { address, link, size, scale, hasTooltip, isUser } = this.props
+    const {
+      address,
+      link,
+      size,
+      scale,
+      hasTooltip,
+      isUser,
+      className
+    } = this.props
 
     return (
       <div
-        className="AddressLink"
+        className={`AddressLink ${className}`}
```

Comment:
Shouldn't we remove this Component from the `HomePage.js` page?

Reply:
I will remove it on the next PR, where I get rid of the Sidebar as well and I introduce the new top navbar (;